packages/jsii-pacmak/lib/targets/dotnet/dotnetgenerator.ts

```diff
@@ -272,9 +272,16 @@ export class DotNetGenerator extends Generator {
             this.code.line();
         }
 
+        if (initializer) {
+            this.dotnetRuntimeGenerator.emitDeprecatedAttributeIfNecessary(initializer);
+        }
         this.code.openBlock(`protected ${className}(ByRefValue reference): base(reference)`);
```

Comment:
Can't properties and methods also be "deprecated" instead of just at the class level?

Reply:
It's handled when emitting the attributes.

Here is an example for methods: https://github.com/awslabs/jsii/blob/677645dc734dc5f2abb059c05eb295aa939524eb/packages/jsii-pacmak/lib/targets/dotnet/dotnetruntimegenerator.ts#L74